flake8_django/checkers/base_model_checker.py

```diff
@@ -10,6 +10,20 @@ class BaseModelChecker(Checker):
 
     model_name_lookup = None
 
+    def is_abstract_model(self, base):
+        # look for "class Meta"
+        for elem in base.body:
+            if isinstance(elem, ast.ClassDef) and elem.name == 'Meta':
+                # look for "abstract = True"
+                for inner in elem.body:
+                    if (
+                            isinstance(inner, ast.Assign)
+                            and inner.value.value
+                            and any(t.id == 'abstract' for t in inner.targets)
+                    ):
```

Comment:
@denizdogan what do you think of moving this checks to another method to simplify the code on this one:

```python
   def _is_abstract_and_set_to_true(self, element):
        return (
            isinstance(element, ast.Assign)
            and element.value.value is True
            and any(target.id == 'abstract' for target in element.targets)
        )

    def is_abstract_model(self, base):
        # look for "class Meta"
        for element in base.body:
            if isinstance(element, ast.ClassDef) and element.name == 'Meta':
                # look for "abstract = True"
                for inner_element in element.body:
                    if self._is_abstract_and_set_to_true(inner_element):
                        return True
        return False
```

Also notice that I've:

- Modified the check of `element.value.value` to `element.value.value is True` just to be more explicit about the value of it.
- Renamed the variables `elem` to `element, `inner` to `inner_element`, and `t` to `target`.

Could you also add a docstring to both this methods? Thanks!